html/mycakeapp/src/Controller/UsersController.php

```diff
@@ -74,7 +74,7 @@ public function logout()
     public function beforeFilter(Event $event)
     {
         parent::beforeFilter($event);
-        $this->Auth->allow(['login', 'index', 'add']);
+        $this->Auth->allow(['login']);
```

Comment:
```suggestion
$this->Auth->allow(['login', 'index', 'add']);
```
usersのアクセス権は変更しないでください～
講師がレビューするためのユーザを追加しやすくしています！

Reply:
失礼いたしました。教科書に合わせて変更してしまっていました。。